contrib/golang/filters/http/source/go/pkg/api/type.go

```diff
@@ -83,6 +83,10 @@ type HeaderMap interface {
 	// If multiple values associated with this key, first one will be returned.
 	Get(key string) (string, bool)
 
+	// Values returns all values associated with the given key.
+	// The returned slice is not a copy.
+	Values(key string) []string
```

Comment:
nit: will `GetAll` be better here?

Reply:
This name is borrowed from Go's stdlib: https://pkg.go.dev/net/http#Header.Values, so it can be familiar to the Go developers.